clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/xml/ClusteringCacheManagerServiceConfigurationParser.java

```diff
@@ -0,0 +1,466 @@
+                } else {
+                  String portString = port.getValue();
+                  address = InetSocketAddress.createUnresolved(host, Integer.parseInt(portString));
+                }
+                serverAddresses.add(address);
+              }
+
+              break;
+            case "read-timeout":
+              /*
+               * <read-timeout> is an optional element
+               */
+              getTimeout = processTimeout(fragment, item);
+
+              break;
+            case "write-timeout":
+              /*
+               * <write-timeout> is an optional element
+               */
+              putTimeout = processTimeout(fragment, item);
+
+              break;
+            case "connection-timeout":
+              /*
+               * <connection-timeout> is an optional element
+               */
+              connectionTimeout = processTimeout(fragment, item);
+
+              break;
+            case "server-side-config":
+              /*
+               * <server-side-config> is an optional element
+               */
+              serverConfig = processServerSideConfig(item);
+              break;
+            default:
+              throw new XmlConfigurationException(
+                String.format("Unknown XML configuration element <%s> in <%s>",
+                  item.getNodeName(), fragment.getTagName()));
+          }
+        }
+      }
+
+      try {
+        Timeouts timeouts = getTimeouts(getTimeout, putTimeout, connectionTimeout);
+        if (serverConfig == null) {
+          if (connectionUri != null) {
+            return new ClusteringServiceConfiguration(connectionUri, timeouts);
+          } else {
+            return new ClusteringServiceConfiguration(serverAddresses, clusterTierManager, timeouts);
+          }
+        }
+
+        ServerSideConfiguration serverSideConfiguration;
+        if (serverConfig.defaultServerResource == null) {
+          serverSideConfiguration = new ServerSideConfiguration(serverConfig.pools);
+        } else {
+          serverSideConfiguration = new ServerSideConfiguration(serverConfig.defaultServerResource, serverConfig.pools);
+        }
+
+        if (connectionUri != null) {
+          return new ClusteringServiceConfiguration(connectionUri, timeouts, serverConfig.autoCreate, serverSideConfiguration);
+        } else {
+          return new ClusteringServiceConfiguration(serverAddresses, clusterTierManager, timeouts, serverConfig.autoCreate, serverSideConfiguration);
+        }
+      } catch (IllegalArgumentException e) {
+        throw new XmlConfigurationException(e);
+      }
+    }
+    throw new XmlConfigurationException(String.format("XML configuration element <%s> in <%s> is not supported",
+      fragment.getTagName(), (fragment.getParentNode() == null ? "null" : fragment.getParentNode().getLocalName())));
+  }
+
+  @Override
+  public Class<ClusteringService> getServiceCreationType() {
```

Comment:
It's not a service creation type... it's a service type.

Reply:
I will make the change